.buildkite/pipeline.yml

```diff
@@ -0,0 +1,6 @@
+steps:
```

Comment:
Did we previously define this in the Buildkite CI? Have you updated Buildkite to expect a YAML definition in the repo?

Or where did the steps previously come from?

Reply:
I updated buildkite to expect a yaml definition. The steps were previously defined through the legacy (pre-yaml) step definition UI.